control-plane-operator/controllers/hostedcontrolplane/cvo/params.go

```diff
@@ -28,7 +28,7 @@ func NewCVOParams(hcp *hyperv1.HostedControlPlane, releaseImageProvider *imagepr
 		CLIImage:                releaseImageProvider.GetImage("cli"),
 		AvailabilityProberImage: releaseImageProvider.GetImage(util.AvailabilityProberImageName),
 		ControlPlaneImage:       util.HCPControlPlaneReleaseImage(hcp),
-		ReleaseImage:            hcp.Spec.ReleaseImage,
+		ReleaseImage:            releaseImageProvider.GetImage("hypershift-cluster-version-operator"),
```

Comment:
Just a couple more things:
- can we just rename this to "cluster-version-operator" ?
- could we fall back to hcp.Spec.ReleaseImage if we cannot get the one from the releaseImageProvider? (I'm thinking of local development use case where we may not be setting the env var)

Reply:
updated.